### ISSUE TITLE
Support revocations for non ethr-did credentials

### ISSUE THREAD

**Issue by ugoamanoh** [uport-project/kotlin-did-jwt#38]
Require and use a DIDDocument param for checkStatus gather valid revokers (support revocations for non ethr-did)

**Comment by ugoamanoh** [comment]
**Implementation Notes**

```
- get credential

- get issuer

- resolve to get did document

- verify that issuer & controller match

- get ethereum addresses from public keys

- create a list of valid revokers (if ethr-did, add issuer)

- run credential check against valid revokers
```

**Comment by mirceanis** [comment]
closed in #42
